### DISCUSSION TITLE
Nashville Number System

### DISCUSSION THREAD

**Discussion by hfl1967** [PetrIvan/chord-seq-ai-app#50] [Feature requests]
It would be helpful to add in the Nashville Number System as a toggle or in the bottom corner of the chord block. It's just one more way to learn more about progressions. Thanks for consideration!

**Comment by PetrIvan** [comment]
Hi, implementing the Nashville Number System (or almost equivalently Roman numerals) could indeed be useful, although I am not sure how to keep track of the tonic. The first chord often isn't the tonic, detection based on the scale of a few chords can be ambiguous, and modulations complicate it even further. The best option in my opinion would be to specify the tonic manually and treat any modulated parts as if they were still in the previous key. The visualization type together with the tonic could be available by opening some settings UI. Let me know what you think!
